### DISCUSSION TITLE
Copy / Duplicate Organization

### DISCUSSION THREAD

**Discussion by tanto-satu** [growthbook/growthbook#2864] [Q&A] [answered]
Hi, just want to ask can we duplicate or copy feature toggles across organization? or is there any way to implement this using APIs?

**Comment by alyssamichelle** [comment, marked as answer]
There isn’t currently a built-in way to duplicate or copy feature flags across organizations in GrowthBook.

Within the same org, you can copy rules between environments or fork an environment, but neither of those works across orgs.

For cross-org or cross-instance copies, the best option today is a small script using the REST API to pull the feature definitions from the source and recreate them in the destination.

There’s also an open feature request for a native export/import flow:
https://github.com/growthbook/growthbook/issues/443 I will go ahead and make a note about this request. Thank you for the discussion!
